.env.schema

```diff
@@ -26,8 +26,9 @@ NEXT_PUBLIC_DOWNLOAD_ALL_URL=
 NEXT_PUBLIC_RELEASE_DATE=
 
 # ######## Covizu
-NEXT_PUBLIC_COVIZU_DATA_VERSION=
+NEXT_PUBLIC_COVIZU_VERSION=
```

Comment:
renamed dataVersion to covizuVersion because it now only indicates the *code* version, not the data version.